docs/tutorials/fabric_remote.md

```diff
@@ -0,0 +1,425 @@
+---
+layout: default
+title: Work with remote fabric network
+parent: pages.tutorials
+nav_order:
+---
+
+# Work with remote Hyperledger Fabric Network
+
```

Comment:
```suggestion
```
Removing this extra line of whitespace will achieve the desired effect of not having the page title in the page's table of contents below